firebase/rules/tests/executions.spec.ts

```diff
@@ -77,4 +82,15 @@ describe('/executions', () => {
       expect(systemUser).cannotWrite('/executions/1/messages/kind', string51);
     });
   });
+
+  it('Only system users can write into common lab property of executions', () => {
```

Comment:
Nitpick:

>Only system user...

There really only is one system user account. It's not like a whole category of users but really only one single special account.